1.5.1.0 from fdroid crashes on startup

**App version:** 1.5.1.0 from F-droid

**Android version**: 6.0.1, cyanogenmod cm-13.0-20160216-nightly

**Devide model**: lg g4


Today I updated from 1.4 something, and it immediately crashes on startup, "Unfortunately, AntennaPod has stopped."

I tried installing the 2 older versions listed in fdroid, both times it failed with "unspecified error." This is using the latest fdroid.

[ Environment ]
Android version: 6.0.1
OS version: 3.10.84-g6d78ce6
AntennaPod version: 1.5.1.0
Model: LG-H811
Device: h811
Product: g4_global_com

[ StackTrace ]
java.lang.NoClassDefFoundError: Failed resolution of: Landroid/support/v7/widget/TintManager;
	at android.support.design.widget.TabLayout$TabView.<init>(TabLayout.java:1185)
	at android.support.design.widget.TabLayout.setTabsFromPagerAdapter(TabLayout.java:8656)
	at android.support.design.widget.TabLayout.setupWithViewPager(TabLayout.java:616)
	at de.danoeh.antennapod.fragment.EpisodesFragment.onCreateView(EpisodesFragment.java:54)
	at android.support.v4.app.Fragment.performCreateView(Fragment.java:1974)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1252)
	at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:738)
	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1617)
	at android.support.v4.app.FragmentController.execPendingActions(FragmentController.java:339)
	at android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:602)
	at de.danoeh.antennapod.activity.MainActivity.onStart(MainActivity.java:450)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1238)
	at android.app.Activity.performStart(Activity.java:6268)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2413)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2520)
	at android.app.ActivityThread.-wrap11(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1363)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:148)
	at android.app.ActivityThread.main(ActivityThread.java:5466)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.v7.widget.TintManager" on path: DexPathList[[zip file "/data/app/de.danoeh.antennapod-1/base.apk"],nativeLibraryDirectories=[/data/app/de.danoeh.antennapod-1/lib/arm64, /vendor/lib64, /system/lib64]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
	... 24 more
	Suppressed: java.lang.ClassNotFoundException: android.support.v7.widget.TintManager
		at java.lang.Class.classForName(Native Method)
		at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
		at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
		... 25 more
	Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack trace available

```
[ Environment ]
Android version: 5.1.1
OS version: 3.4.67-cyanogenmod-g9f92a21
AntennaPod version: 1.5.1.0
Model: A0001
Device: A0001
Product: bacon

[ StackTrace ]
java.lang.NoClassDefFoundError: Failed resolution of: Landroid/support/v7/widget/TintManager;
	at android.support.design.widget.TabLayout$TabView.<init>(TabLayout.java:1185)
	at android.support.design.widget.TabLayout.setTabsFromPagerAdapter(TabLayout.java:8656)
	at android.support.design.widget.TabLayout.setupWithViewPager(TabLayout.java:616)
	at de.danoeh.antennapod.fragment.DownloadsFragment.onCreateView(DownloadsFragment.java:47)
	at android.support.v4.app.Fragment.performCreateView(Fragment.java:1974)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1252)
	at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:738)
	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1617)
	at android.support.v4.app.FragmentController.execPendingActions(FragmentController.java:339)
	at android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:602)
	at de.danoeh.antennapod.activity.MainActivity.onStart(MainActivity.java:450)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1236)
	at android.app.Activity.performStart(Activity.java:6006)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2321)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2420)
	at android.app.ActivityThread.access$900(ActivityThread.java:154)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1321)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5294)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.v7.widget.TintManager" on path: DexPathList[[zip file "/data/app/de.danoeh.antennapod-2/base.apk"],nativeLibraryDirectories=[/vendor/lib, /system/lib]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
	... 25 more
	Suppressed: java.lang.ClassNotFoundException: android.support.v7.widget.TintManager
		at java.lang.Class.classForName(Native Method)
		at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
		at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
		... 26 more
	Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack available

```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -71,6 +71,7 @@ android {
         versionName "${getMyVersionName()}"
         testApplicationId "de.test.antennapod"
         testInstrumentationRunner "de.test.antennapod.AntennaPodTestRunner"
+        generatedDensities = []
     }
 
     signingConfigs {
@@ -131,6 +132,10 @@ android {
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    aaptOptions {
+        additionalParameters "--no-version-vectors"
     }
 }
 

--- a/build.gradle
+++ b/build.gradle
@@ -41,7 +41,7 @@ project.ext {
     minSdkVersion = 10
     targetSdkVersion = 23
 
-    supportVersion = "23.1.1"
+    supportVersion = "23.2.0"
     commonsioVersion = "2.4"
     commonslangVersion = "3.4"
     eventbusVersion = "2.4.0"
@@ -49,7 +49,7 @@ project.ext {
     glideVersion = "3.6.1"
     iconifyVersion = "2.1.1"
     jsoupVersion = "1.7.3"
-    materialDialogsVersion = "0.8.5.3@aar"
+    materialDialogsVersion = "0.8.5.6@aar"
     okhttpVersion = "2.7.4"
     okioVersion = "1.6.0"
     recyclerviewFlexibledividerVersion = "1.2.6"

--- a/core/src/main/res/values/styles.xml
+++ b/core/src/main/res/values/styles.xml
@@ -63,6 +63,7 @@
 
     <style name="Theme.AntennaPod.Dark" parent="Theme.AppCompat">
         <item name="colorAccent">@color/holo_blue_dark</item>
+        <item name="colorControlNormal">@color/white</item>
         <item name="buttonStyle">@style/Widget.AntennaPod.Button</item>
         <item name="progressBarTheme">@style/ProgressBarDark</item>
         <item name="alertDialogTheme">@style/AntennaPod.Dialog.Dark</item>
@@ -186,6 +187,7 @@
         <item name="windowActionModeOverlay">true</item>
         <item name="progressBarTheme">@style/ProgressBarDark</item>
         <item name="colorAccent">@color/holo_blue_dark</item>
+        <item name="colorControlNormal">@color/white</item>
         <item name="buttonStyle">@style/Widget.AntennaPod.Button</item>
         <item name="alertDialogTheme">@style/AntennaPod.Dialog.Dark</item>
         <item name="attr/action_about">@drawable/ic_info_white_24dp</item>

